src/library/scala/collection/package.scala

```diff
@@ -94,10 +96,16 @@ package object collection {
    */
   def breakOut[From, T, To](implicit b: CanBuildFrom[Nothing, T, To]): CanBuildFrom[From, T, To] =
     // can't just return b because the argument to apply could be cast to From in b
-    new CanBuildFrom[From, T, To] {
-      def apply(from: From) = b.apply()
-      def apply()           = b.apply()
-    }
+    new WrappedCanBuildFrom[From, T, To](b)
+
+  private[collection] final class WrappedCanBuildFrom[From, T, To](
+        wrapped: CanBuildFrom[Nothing, T, To])
+    extends CanBuildFrom[From, T, To] {
+    def apply(from: From) = wrapped.apply()
+
+    def apply() = wrapped.apply()
+    final def unwrap:  CanBuildFrom[_, T, To] = wrapped
```

Comment:
Why do we need this method that drops the `From` type instead of making `wrapped` a `val` and keeping `Nothing`?

Reply:
no idea. Changed it to be a val